src/vos/vos_dtx.c

```diff
@@ -41,6 +41,48 @@ D_CASSERT((DTX_BLOB_SIZE / sizeof(struct vos_dtx_cmt_ent_df)) <  (1 << 15));
 #define DTX_ACT_BLOB_MAGIC	0x14130a2b
 #define DTX_CMT_BLOB_MAGIC	0x2502191c
 
+enum {
+	DTX_UMOFF_ILOG		= (1 << 0),
+	DTX_UMOFF_SVT		= (1 << 1),
+	DTX_UMOFF_EVT		= (1 << 2),
+};
```

Comment:
The description says the high 8 bits but this is the low 3 bits.   I am wondering why our flags are using the high 8 bits as that won't work for virtual memory.  I don't recall why I did it that way.   It seems everthing would be allocated to 8 byte alignment so the lower 3 bits is safe.

Reply:
Originally, one of the usage for the high 8-bits is to indicate it is an aborted entry. But such usage is removed by your patch PR#2634.

In my patch, the "type" use 3 of the high 8-bits. They are set as following:
umem_off_set_flags(rec, dtx_type2umoff_flag(type));

But I think it is better to call the umem_off_set_flags() inside dtx_type2umoff_flag() to make things more clear. I will adjust the patch for that.